taxonomies/food/categories.txt

```diff
@@ -49460,16 +49460,22 @@ ciqual_food_code:en: 19666
 ciqual_food_name:en: Petit-Suisse like fresh cream cheese, plain, around 9% fat
 ciqual_food_name:fr: Fromage frais type petit suisse, nature, 10% MG environ
 
+# Half milk and half cream (or Half and half) is a common preparation in the USA which is used in coffee or with breakfast cereals
+< en:Dairies
+en: Half milk and half cream
```

Comment:
Could you add "Half and half" as a synonym?

Reply:
I added it!

Reply:
Thank you!